<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSmootherWindow</class>
 <widget class="QMainWindow" name="ImageSmootherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="viewFrom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="viewTo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sigmaSlider">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sigmaValue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
